<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(39, 39, 39);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTextEdit" name="textEdit_Campo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>561</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
color: rgb(255, 255, 255);
border-radius: 15px;
border-color: rgb(255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_analizarCodigo">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>720</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 219, 25);
color: rgb(0, 0, 0);
border-radius: 15px;
font: 63 8pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>ANALIZAR CÓDIGO</string>
    </property>
   </widget>
   <widget class="QToolButton" name="pushButton_buscarArchivo">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 63 8pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Selecccionar archivo:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cargarArchivo">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 54, 3);
color: rgb(255, 255, 255);
border-radius: 15px;
font: 63 8pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Cargar archivo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textField_selected_file">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_alert">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Import a file!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_file_sucessful">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 0);</string>
    </property>
    <property name="text">
     <string>File Sucessful!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_limpiarCodigo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>720</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(242, 219, 25);
border-radius: 15px;
font: 63 8pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>LIMPIAR CÓDIGO</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabla_tokens">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>381</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 255, 255);</string>
    </property>
    <column>
     <property name="text">
      <string>TOKENS</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>VALORES</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>POSICION</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 20pt &quot;Rockwell Extra Bold&quot;;
color: rgb(247, 224, 24);</string>
    </property>
    <property name="text">
     <string>JAVASCRIPT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_erroresLexico">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>720</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 0, 0);
color: rgb(255, 255, 255);
border-radius: 15px;
font: 63 8pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>VER ERRORES DE LÉXICO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lexicoValido">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>280</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 0);
font: 63 12pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>LÉXICO VÁLIDO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_sintaticoValido">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>360</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 0);
font: 63 12pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>SINTACTICO VÁLIDO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_lexicoInvalido">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>280</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 63 12pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>LÉXICO INVÁLIDO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_sintaticoInvalido">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>360</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 63 12pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>SINTACTICO INVÁLIDO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
